doc/Getting-started/index.md

```diff
@@ -65,543 +41,211 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
-For programming practice, see [messaging across endpoints]().
+# Troubleshooting
 
-**Collecting data**
-Kaa provides rich capabilities for collecting and storing structured data from endpoints. A typical use-case is collecting various types of logs: performance, user behavior, exceptional conditions, etc.
-
-Using a set of pre-packaged server-side _log appenders_, the Kaa server is able to store records to a filesystem, a variety of big data platforms (Hadoop, MongoDB, Cassandra, Oracle NoSQL etc.), or submit them directly to a streaming analytics system. It is also possible to [create a custom log appender***]().
-
-The structure of the collected data is flexible and defined by the [log schema***](). Based on the log schema defined for the Kaa application, Kaa generates an object model for the records and the corresponding API calls in the client SDK. Kaa also takes care of data marshalling, managing temporary data storage on the endpoint, and uploading data to the Kaa server.
-
-For programming practice, see [collecting data from endpoints***]().
-
-**Using notifications**
-Kaa uses _notifications_ to distribute structured messages, posted within _notification topics_, from the server to endpoints. A notification structure is defined by a corresponding [notification schema***]().
-
-Endpoint are subscribed to notification topics, which can be either mandatory or optional. Access to notification topics is automatically granted according to the endpoint’s group membership. Notifications can be sent either to every endpoint subscribed to a topic or to an individual endpoint.
-
-Notifications can be assigned expiration timestamps to prevent their delivery after a certain period of time.
-
-For programming practice, see [using notifications***]().
-
-**Distributing operational data**
-Kaa allows you to perform operational data updates, such as configuration data updates, from the Kaa server to endpoints. This feature can be used for centralized configuration management, content distribution, etc. Since Kaa works with structured data and constraint types, it guarantees data integrity.
+Common issues covered in this [guide]({{root_url}}Administration-guide/Troubleshooting/).
```

Comment:
As a general rule, don't say `this page`, `this guide`, etc. Give a name of the page, so the user knows what he's opening.